lib/BackupPC/CGI/Metrics.pm

```diff
@@ -155,17 +157,19 @@ sub action
         }
 
         $metrics{hosts}{$host} = {
+            full_age          => int($fullAge),
             full_start_time   => int($$lastFullBackup->{startTime}),
             full_count        => $fullCount,
             full_duration     => int($fullDuration),
-            full_keep_count   => $Conf{FullKeepCnt},
+            full_keep_count   => $Conf{FullKeepCnt}[0],
```

Comment:
I think this `[0]` is no longer needed.

I was getting an error until I removed it.

Reply:
Check please